### PR TITLE
feat: add tag latest to the mediator docker image

### PR DESCRIPTION
**Pull request by FabioPinheiro**

### CODE REVIEW

**Review by elribonazo — APPROVED** [review]

**Review by antonbaliasnikov — COMMENTED** [review]
This will not work. `dockerUpdateLatest` has not impact, as we just generate docker file in the process, but we actually building an image in `package.json` file

**Comment by FabioPinheiro** [comment]
yeah this is just when we publish locally

**Review by elribonazo — APPROVED** [review]

**Review by ghost — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -131,6 +131,17 @@ To set up the mediator storage (MongoDB):
...
 
+## Run
+
+This DIDComm Mediator is composed of two elements, a backend service, and a database.
+The backend service is a JVM application and the database used is MongoDB.
+The backend service is also a web service that has a single-page application that will give the final user an invitation page.
+
+### Run localy
+
+Everything can be run with a single command with Docker compose `docker-compose up`.
```
Comment (petevielhaber) [inline README.md:142]: Everything runs with a single Docker compose command:

File: README.md [hunk 2]
```diff
@@ -131,6 +131,17 @@ To set up the mediator storage (MongoDB):
...
+For a specific version you can setup with `MEDIATOR_VERSION` like `MEDIATOR_VERSION=0.9.2-SNAPSHOT docker-compose up`.
```
Comment (petevielhaber) [inline README.md:143]: You can set up a specific version with `MEDIATOR_VERSION` like `MEDIATOR_VERSION=0.9.2-SNAPSHOT docker-compose up`.

File: README.md [hunk 3]
```diff
@@ -131,6 +131,17 @@ To set up the mediator storage (MongoDB):
 - `MONGODB_PASSWORD` - is the password used by the Mediator service to connect to the database.
 - `MONGODB_DB_NAME` - is the name of the database used by the Mediator.
 
+## Run
+
+This DIDComm Mediator is composed of two elements, a backend service, and a database.
+The backend service is a JVM application and the database used is MongoDB.
+The backend service is also a web service that has a single-page application that will give the final user an invitation page.
```
Comment (petevielhaber) [inline README.md:138]: The DIDComm Mediator comprises two elements: a backend service and a database.

The backend service is a JVM application, and the database used is MongoDB.

The backend service is also a web service with a single-page application that will give the final user an invitation page.

File: README.md [hunk 4]
```diff
@@ -131,6 +131,17 @@ To set up the mediator storage (MongoDB):
...
+
+### Run localy
+
+Everything can be run with a single command with Docker compose `docker-compose up`.
+For a specific version you can setup with `MEDIATOR_VERSION` like `MEDIATOR_VERSION=0.9.2-SNAPSHOT docker-compose up`.
+
 ### MongoDB In cloud
 
 Using the mongodb from cloud like MongoDB Atlas.
```
Comment (petevielhaber) [inline README.md:147]: Use the Mongodb from the cloud like MongoDB Atlas.

**Review by petevielhaber — COMMENTED** [review]
- Suggested changes

**Pull request merged**